extra/exports

```diff
@@ -86,6 +87,8 @@ box_latch_new
 box_latch_trylock
 box_latch_unlock
 box_on_shutdown
+box_read_ffi_disable
+box_read_ffi_enable
```

Comment:
Nit: `box_read_ffi_enable` is never used from Lua so it shouldn't be exported.

Reply:
Done